packages/kbn-visualization-ui-components/components/chart_switch_trigger.tsx

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
@MichaelMarcialis thank you for your comments! I'll keep the responses here so we can take advantage of @markov00 proposal to keep the context in one place. First issue posted by you:

> The configuration toolbar currently scrolls with any overflowing layer panel(s). Would it be possible to restructure the markup or CSS in a way that would limit the scrolling container to the layer panel(s) only, thus allowing the toolbar to always be visible to the user?

Reply:
I am not sure it's a way to go. Here's how it looks right now in my Macbook 16'. The scrolling area is already extremely small for ESQL charts and making it even smaller makes it not even fit one layer configuration. I think it makes it worse for the user and I'd prefer to keep scroll as it is. Whether we decide to change it or not, I can add an issue and keep it separate from this PR.

![Mar-07-2024 13-40-51](https://github.com/elastic/kibana/assets/4283304/e7bef4d4-b56b-4946-a6a5-ab075550e9b1)

Reply:
That's a fair point. Yeah, let's split it off as a separate issue to tackle, since it will likely require more design thought and possible layout shuffling (beyond the scope of this PR). Thanks!

Reply:
Done here: https://github.com/elastic/kibana/issues/178234